web-api/src/applicationContext.js

```diff
@@ -711,6 +712,7 @@ module.exports = (appContextUser, logger = createLogger()) => {
         getDocumentTypeForAddressChange,
         getFormattedCaseDetail,
         getStampBoxCoordinates,
+        getTrialSessionStatus,
```

Comment:
feels a little persnickety, but I think this might better belong inside `shared/src/business/dto/TrialSessionInfoDto.ts` or a . Happy for this to be a future refactoring task that really targets trimming down functions in `getUtilities()` into places where they make better sense. It just feels like a catch all toolbox at the moment. What do you think?

Reply:
sounds ok since this helper function isn't used in the frontend and only used from other business layer functions / entities.